OtripleS.Web.Api/Migrations/20200907151159_RemovedGuardianRelationShipModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace OtripleS.Web.Api.Migrations
+{
+    public partial class RemovedGuardianRelationShipModel : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "Relationship",
+                table: "Guardians");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "Relationship",
+                table: "Guardians",
+                type: "int",
+                nullable: false,
+                defaultValue: 0);
+        }
+    }
+}
```

Comment:
I suggest you remove all the migrations you have created for this entity and start over after you've modified your model.
Otherwise it will track changes we don't really care to revert to at any point in time.

You can revert your migrations by simply doing this:
1. disable auto-migration - you can just temporarily comment this line:
https://github.com/hassanhabib/OtripleS/blob/master/OtripleS.Web.Api/Brokers/Storage/StorageBroker.cs#L22

2. Unapply all migrations by running this command:
```powershell
Update-Database -Migration 0
```

3. remove your last migration (maybe two since you've done a couple of those now in this PR):
```powershell
Remove-Migration
```

4. start your migrations over with the new model.

Let me know if you have any questions.

Reply:
Applied and Created Migration with a new model